src/firefly/js/tables/FilterInfo.js

```diff
@@ -22,14 +25,30 @@ export const FILTER_TTIPS =
 * string value must be enclosed in single quotes
 * when IN is used, enclose the values in parentheses
 * you may combine conditions with either 'and' or 'or'
-* grouping by parentheses is not supported at the moment 
-Examples:  "ra" > 12345 and "color" != 'blue' or "band" IN (1,2,3)
+* grouping by parentheses is not supported at the moment
+* '${FILTER_SEP}' serves as groups separator at the moment 
+Examples:  
+  "ra" > 5 and "color" != 'blue' or "band" IN (1,2,3)
+  "ra" < 5 or "ra" > 6 _AND_ "dec" < -1 or "dec" > 1
```

Comment:
I see why this is hard.  But, I really prefer we don't expose _AND_.  It's intended for `internal` use as commented above.

Reply:
I've removed mention of the _AND_ from the tooltip. I agree, that the fact that Filters currently show our internal representation of filters is not ideal, but currently it is the only way we have to clear or change non-table filters selectively. 
I think clearing this logic should be in a separate ticket. It's not a quick bug fix.